eng/packaging.targets

```diff
@@ -55,42 +96,62 @@
+    </PropertyGroup>
+    <ItemGroup Condition="'$(_documentationFileToPackage)' != ''">
+      <!-- Place the documentation file into the lib or runtimes/lib folder depending on if the build is runtime agnostic or specific. -->
+      <TfmSpecificPackageFile Include="$(_documentationFileToPackage)"
+                              PackagePath="$([MSBuild]::ValueOrDefault('$(BuildOutputTargetFolder)', 'lib'))/$(TargetFrameworkWithoutSuffix)"
+                              Condition="'$(TargetsAnyOS)' == 'true'" />
+      <TfmRuntimeSpecificPackageFile Include="$(_documentationFileToPackage)"
+                                     Condition="'$(TargetsAnyOS)' != 'true'" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="AddRuntimeSpecificFilesToPackage"
+          DependsOnTargets="BuiltProjectOutputGroup;
+                            SatelliteDllsProjectOutputGroup;
+                            $(TargetsForTfmSpecificBuildOutput);
+                            AddDocumentationFileToPackage"
+          Condition="'$(ExcludeFromPackage)' != 'true' and
+                     '$(TargetsAnyOS)' != 'true'">
     <ItemGroup>
-      <TfmSpecificPackageFile Include="@(SatelliteDllsProjectOutputGroupOutput);
-                            @(BuiltProjectOutputGroupOutput)">
-        <PackagePath>runtimes/$(_runtimeOS)/lib/$(TargetFrameworkWithoutSuffix)</PackagePath>
-      </TfmSpecificPackageFile>
+      <TfmRuntimeSpecificPackageFile Include="@(BuiltProjectOutputGroupOutput);
+                                              @(SatelliteDllsProjectOutputGroupOutput)" />
+      <TfmSpecificPackageFile Include="@(TfmRuntimeSpecificPackageFile)"
+                              PackagePath="runtimes/$(PackageTargetRuntime)/lib/$(TargetFrameworkWithoutSuffix)" />
+      <!-- Copy runtime specific assemblies into the rid agnostic 'lib' folder if the ridless tfm doesn't exist on .NETFramework.
+           The assumption holds that a ridless tfm doesn't follow a '-' charater. This is necessary to avoid NU5128. -->
+      <TfmSpecificPackageFile Include="@(TfmRuntimeSpecificPackageFile)"
+                              PackagePath="$([MSBuild]::ValueOrDefault('$(BuildOutputTargetFolder)', 'lib'))/$(TargetFrameworkWithoutSuffix)"
+                              Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework' and !$([System.Text.RegularExpressions.Regex]::IsMatch('$(TargetFrameworks)', '$(TargetFrameworkWithoutSuffix)(?!-)'))" />
     </ItemGroup>
   </Target>
 
-  <!-- Runtime independent symbols are automatically added by the pack task.-->
-  <Target Name="AddRuntimeDebugSymbolsWithTfm"
-          Condition="'$(IncludeSymbols)' == 'true' and '$(TargetsAnyOS)' != 'true' and '$(ExcludeFromPackage)' == ''">
+  <!-- Runtime agnostic symbols are automatically added by the pack task.-->
+  <Target Name="AddRuntimeSpecificSymbolToPackage"
```

Comment:
It's a different extension point "TargetsForTfmSpecificDebugSymbolsInPackage" than "TargetsForTfmSpecificContentInPackage" so IMO it makes sense to be a separate target.

Reply:
You could have a single target and include it in both lists, it'd only run once.  Up to you what you think is more maintainable.  Seemed to me like it might be nice to have symbol inclusion next to binary inclusion.